model/anomaly_nn.ipynb

```diff
@@ -1,991 +1,1322 @@
-      "text/plain": [
-       "<Figure size 432x288 with 1 Axes>"
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "hbvFNaRk-yU2"
+      },
+      "source": [
+        "## Setup Model"
       ]
-     },
-     "metadata": {
-      "needs_background": "light"
-     },
-     "output_type": "display_data"
-    }
-   ],
-   "source": [
-    "# plot cost functions\n",
-    "minibatch_cost_cpu = [i.cpu().detach().numpy() for i in minibatch_cost]\n",
-    "# epoch_cost_cpu = [i.cpu().detach().numpy() for i in epoch_performance]\n",
-    "\n",
-    "\n",
-    "plt.plot(range(len(minibatch_cost_cpu)), minibatch_cost_cpu)\n",
-    "plt.ylabel('Cost Function Label') # update this label after cost function selected\n",
-    "plt.xlabel('Minibatch')\n",
-    "plt.show()\n",
-    "\n",
-    "plt.plot(range(len(epoch_train_performance)), epoch_train_performance, label=\"train f1 scores\")\n",
-    "plt.plot(range(len(epoch_val_performance)), epoch_val_performance, label=\"val f1 scores\")\n",
-    "plt.ylabel('Accuracy') # update this label after cost function selected\n",
-    "plt.xlabel('Epoch')\n",
-    "plt.legend(bbox_to_anchor=(1.05, 1), loc='upper left', borderaxespad=0.)\n",
-    "plt.show()"
-   ]
-  },
-  {
-   "cell_type": "markdown",
-   "metadata": {},
-   "source": [
-    "## Evaluation"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 413,
-   "metadata": {
-    "tags": [
-     "outputPrepend"
-    ]
-   },
-   "outputs": [
+    },
     {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Test f1_score: 0.43%\n"
-     ]
-    }
-   ],
-   "source": [
-    "with torch.set_grad_enabled(False): # save memory during inference\n",
-    "    # THIS NEEDS TO BE UPDATED BASED ON THE FUNCTION NAME WE'RE USING\n",
-    "    print('Test f1_score: %.2f%%' % (compute_f1(model, test_loader, device=DEVICE)))"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 414,
-   "metadata": {},
-   "outputs": [
+      "cell_type": "code",
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "KFxoCAHg-yU2",
+        "outputId": "130bb71b-b963-4527-dfec-bc3b8514ed9d"
+      },
+      "source": [
+        "device = torch.device(DEVICE)\n",
+        "torch.manual_seed(0)\n",
+        "\n",
+        "for epoch in range(2):\n",
```

Comment:
this code block needs comments - that it's just checking the train_loader (as discussed with Debangsha it doesn't have anything to do with warmup